public/angular/translations/locale-en.json

```diff
@@ -19,6 +19,8 @@
   "rss": "RSS",
   "elmo": "Elmo",
   "facebook": "Facebook",
+  "smsgh": "SMS GH",
+  "Smsgh": "SMS GH",
```

Comment:
Search filter for media do call capitalize smsgh
`<select ng-model="searchParams.media" ng-options="media | capitalize | translate for media in mediaOptions | orderBy:'toString()'" ng-change="search()" class="form-control">` it should be changed to `translate`